docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -3,6 +3,13 @@ description: Pantheon JSON-RPC API methods reference
 
 # JSON-RPC API Methods
 
+## Endpoint Host and Port
```

Comment:
Maybe make that more visible using a !!!attention

Reply:
I've updated it.  That is better.